### PR TITLE
docs: add Axon fundamentals

### PR DESCRIPTION
**Pull request by linnnsss**
create a .mdx file for the content, delete the old directory

### CODE REVIEW

**Review by KaoImin — APPROVED** [review]

**Review by linnnsss — COMMENTED** [review]
fix slug

**Review by linnnsss — COMMENTED** [review]
fix slug again

File: docs/axon_fundamentals.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+slug: fundamentals
+title: Axon Fundamentals
+position: 2
+sidebar_position: 2
+---
+
+Axon is a Proof-of-Stake (Pos) framework that enables developers to build app-chains as Layer 2 of CKB network. It provides a secure and efficient way of interacting with multiple participants in the network. Axon relies on a set of key concepts that underpin its operation and the various participants in its economic ecosystem. This essay contains two lists: one summarizes the key concepts; the other outlines the roles of the participants.
+
```
Comment (liya2017) [inline docs/axon_fundamentals.mdx:9]: import useBaseUrl from "@docusaurus/useBaseUrl";

**Review by liya2017 — DISMISSED** [review]

**Review by linnnsss — COMMENTED** [review]
fix image and BaseUrl

**Pull request merged**